staging/src/k8s.io/kubectl/pkg/cmd/diff/diff.go

```diff
@@ -583,6 +590,16 @@ func (d *Differ) Diff(obj Object, printer Printer) error {
 	return nil
 }
 
+func removeManagedFields(obj runtime.Object) {
```

Comment:
Nit: can you do something similar to https://github.com/kubernetes/kubernetes/blob/610b7839a0bc2f9bafaca92930f8dd740f0ab46c/staging/src/k8s.io/cli-runtime/pkg/printers/managedfields.go#L34-L42

Reply:
Whoops, I just saw this comment, will make this change...